<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="1INITCLT02666" ignored="1" name="Personal Loan" tests="12" failures="1" timestamp="15 Feb 2023 03:51:45 GMT" time="0.036" errors="0">
  <testcase name="Demo2" time="0.002" classname="TestPackage.Basics"/>
  <testcase name="Demo4" time="0.002" classname="TestPackage.Basics">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at TestPackage.Basics.Demo4(Basics.java:42)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- Demo4 -->
  <testcase name="Demo6" time="0.002" classname="TestPackage.Basics"/>
  <testcase name="Wemo" time="0.001" classname="TestPackage.Basics"/>
  <testcase name="demo3" time="0.001" classname="TestPackage.Basics2"/>
  <testcase name="dummy" time="0.001" classname="TestPackage.Basics2"/>
  <testcase name="home" time="0.001" classname="TestPackage.Basics2"/>
  <testcase name="personal" time="0.0" classname="TestPackage.Basics2"/>
  <testcase name="demo" time="0.001" classname="TestPackage.Basics5"/>
  <testcase name="demo" time="0.0" classname="TestPackage.Basics5"/>
  <testcase name="demo" time="0.0" classname="TestPackage.Basics5"/>
  <testcase name="Demo3" time="0.0" classname="TestPackage.Basics">
    <skipped/>
  </testcase> <!-- Demo3 -->
  <testcase name="Demo5" time="0.0" classname="TestPackage.Basics">
    <ignored/>
  </testcase> <!-- Demo5 -->
</testsuite> <!-- Personal Loan -->
